apps/docs/content/components/Prose.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Prose
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/Prose/Prose.tsx
+storybook: '/brand/storybook/?path=/story/components-prose--default'
+description: Use the Prose component to render html content with Primer Brand styles.
+---
+
+```js
+import {Prose} from '@primer/react-brand'
+```
+
+## Examples
+
+<Note>
+
+`Prose` is designed to take sanitized HTML as a string and render it with Primer Brand styles. It is not intended to be used with JSX. The `Prose` component doens't sanitize the data internally. It is the responsibility of the consumer to sanitize the data before passing it to the component.
+
+</Note>
+
+### Default
+
+```jsx live
+<Prose
+  rawHtmlMarkup={`
+	<h1>The tools you need to build what you want.</h1>
+    <p>Contribute to projects quickly with automatic environment setup.</p>
+    <p>Make sure you see the changes you care about.</p>
+    <p>Build community around your code.</p>
+
+    <img
```

Comment:
Do we need the start padding here? I think the image should center (per the Blog) or align flush with the text to avoid unexpectedly  offset. cc. @danielguillan.

<img width="980" alt="Screenshot 2023-06-26 at 12 27 39" src="https://github.com/primer/brand/assets/13340707/33c3bf3b-bd4b-49ee-851c-949ea6e66665">

Reply:
I believe this img is only centered but we could left align it. When I did it did look just a bit awkward.